gaia-header.js

```diff
@@ -102,6 +102,7 @@ module.exports = component.register('gaia-header', {
     this.noFontFit = this.getAttribute('no-font-fit');
     this.notFlush = this.hasAttribute('not-flush');
     this.action = this.getAttribute('action');
+    this.ignoreDir = this.hasAttribute('ignore-dir');
```

Comment:
I'd set it before the other attributes, so that the other attributes can count on the property being set